libs/game/controller.ts

```diff
@@ -162,6 +162,8 @@ namespace controller {
         //% vx.defl=100 vy.defl=100
         //% help=controller/move-sprite
         //% group="Multiplayer"
+        //% vx.shadow="speedPicker"
+        //% vy.shadow="speedPicker"
```

Comment:
```suggestion
        //% vx.shadow="spriteSpeedPicker"
        //% vy.shadow="spriteSpeedPicker"
```

I believe to show px/s?